script/example.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+import yaml
+import json
+from argparse import ArgumentParser
+
+def action_json(args):
+
+    with open('scripts/example_data.json') as json_file:
+        print(json.dumps(json.loads(json_file.read()), indent=2))
+
+def action_yaml(args):
+    with open('scripts/example_data.yaml') as yaml_file:
+        print(yaml.dump(yaml.safe_load(yaml_file.read()), indent=2))
+
+def invalid_action(args):
+    print(f'ERROR: Unsupported action: {args.action}')
+    exit(1)
```

Comment:
Using `exit(1)` directly in the function body can make unit testing difficult. Consider raising an exception instead, which can be handled or tested more gracefully.
```suggestion
raise Exception(f'ERROR: Unsupported action: {args.action}')
```